app/validators/salary_validator.rb

```diff
@@ -7,15 +7,16 @@ class SalaryValidator < ActiveModel::EachValidator
 
   SALARY_FORMAT = /^(\d+|\d{1,3}(,\d{3})*)(\.\d{2})?$/.freeze
   MIN_SALARY_ALLOWED = 0
-  MAX_SALARY_ALLOWED = 200000
+  MAX_SALARY_LIMIT = 200000
```

Comment:
Changed this to MAX_SALARY_LIMIT, as one of the desired changes was to make 199,999 the salary limit but the error message would still display 200,000 in the error message. See [here](https://docs.google.com/spreadsheets/d/1-MeFAToarUINg-8Sif4K0ZTPudowE6FU_mf3Hhf_iOk/edit?disco=AAAAD5CLJT4).